admob/src/ios/admob_ios.mm

```diff
@@ -164,17 +164,6 @@ void Terminate() {
 
 const ::firebase::App* GetApp() { return g_app; }
 
-void AdmobInternal::CompleteLoadAdFuture(
```

Comment:
Moved into admob_common as the method definition is the same for Android and iOS.